uint64 timestamp               # time since system startup

uint8 mainbrain_sensor          # is the drone in the water? (sensor mainbrain and sensor power module == 1)
uint8 power_module_sensor        # is the drone on the water? (sensor mainbrain == 0 and sensor power module == 1)
